Mathlib/Computability/Primrec.lean

```diff
@@ -1113,6 +1113,19 @@ theorem list_join : Primrec (@List.join α) :=
     dsimp; induction l <;> simp [*]
 #align primrec.list_join Primrec.list_join
 
+theorem list_bind {f : α → List β} {g : α → β → List σ} (hf : Primrec f) (hg : Primrec₂ g) :
+    Primrec (fun a => (f a).bind (g a)) := list_join.comp (list_map hf hg)
+
+theorem option_toList : Primrec (Option.toList : Option α → List α) :=
```

Comment:
Following the [naming convention](https://leanprover-community.github.io/contribute/naming.html), namespaces should be appended in came case when the unnamespaced name is ambiguous.
```suggestion
theorem optionToList : Primrec (Option.toList : Option α → List α) :=
```